impl/src/main/resources/application.yaml

```diff
@@ -1,10 +1,26 @@
 spring:
   application:
-    name: nutsConsentBridgeAPISpec
+    name: nutsConsentBridge
 
   jackson:
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
 
 server:
   port: 8080
+
+# Health information
+management:
```

Comment:
in Spring Boot you can expose the management interface on a different HTTP port. Can be useful to separate access to management interfaces on network level (since you can also do harmful stuff like stopping the application or making a heap dump).

Is really a production/ops concern though.

Reply:
future work, for know we suggest nothing is exposed for the bridgee

Reply:
also exposing what can be monitored could be a feature or hardcoded set. Not sure what the 'community' wants